dev/dll/Density/Normal.xaml

```diff
@@ -0,0 +1,127 @@
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" 
```

Comment:
"Normal" is not a great name for this. The RS5 values are the new normal. The values you have here are legacy from RS4 and earlier. We don't want to call them "normal" because frankly we would prefer people move forward into the now-current design approved values.

Reply:
Standard.xaml ?

Reply:
"Legacy" is probably more accurate. As of RS5 it's no longer the standard.

Reply:
> "Legacy" is probably more accurate. As of RS5 it's no longer the standard.

Ah so this is just for RS 1-4 then.  I was assuming you were just differentiating from the Compact.xaml file.  So perhaps, Metro,xaml, Standard.xaml, and Compact.xaml?

Reply:
"Legacy.xaml" is the best option IMHO. That's exactly what these are, legacy values. The standard was updated in RS5 to values we provide today for the styling. "Compact.xaml" would be the best for the smallest size we want to provide.